### PR TITLE
Add pt_BR locale.

### PR DESCRIPTION
**Pull request by andresmrm**
Merci beaucoup for your work with this package!

I'm adding Brazilian translation, but I couldn't test it. I tried setting `LANGUAGE_CODE = "pt-br"` in the `example_project`, but it always used `en` locale for this package strings, even when setting to `fr` or `nl`. The other strings in the admin changed to the correct locales. Any idea why?

I also suggest:

- Improving the [docs](https://github.com/Pierre-Sassoulas/django-zxcvbn-password-validator/blob/main/CONTRIBUTING.md) adding these commands (at least I had to run them here):
```shell
python manage.py makemessages --locale=<desired locale>
python manage.py migrate
```
- Setting `LANGUAGE_CODE = "en"` in the `translation_project`.

### CODE REVIEW

File: django_zxcvbn_password_validator/locale/pt_BR/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -0,0 +1,246 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
```
Comment (Pierre-Sassoulas) [inline django_zxcvbn_password_validator/locale/pt_BR/LC_MESSAGES/django.po:4]:
```suggestion
# Portuguese brasilian translations
# Copyright (C) 2024
# This file is distributed under the same license as the django-zxcvbn-password package.
# Andrés Martano  < mail if you wish ?>, 2024.
```

**Review by Pierre-Sassoulas — CHANGES_REQUESTED** [review]
Thank you, nice work ! 

Don't worry about testing it, the po file is valid, it's going to work (with some limitations I don't remember, I guess) :) The translation project was a way to make it testable easily I guess but the settings should be changed for it to works, it's not turnkey.

Feel free to make change to the readme as you see fit, in this MR or in another one. Also you should add yourself at https://github.com/Pierre-Sassoulas/django-zxcvbn-password-validator?tab=readme-ov-file#language-available 😄

**Comment by Pierre-Sassoulas** [comment]
Please rebase on latest main, I fixed the issue with the pipeline on python 3.12 in #95 :)

**Comment by Pierre-Sassoulas** [comment]
Also realized that a new metadata for natural language br_pt should be added in pyproject.toml :)

**Comment by andresmrm** [comment]
Thanks! I think I made all your suggestions. =)

**Review by Pierre-Sassoulas — APPROVED** [review]
Great contribution thank you ! I'm going to release 1.4.4 with it

**Pull request merged**